pipelines/skylab/atac/atac.wdl

```diff
@@ -58,7 +58,7 @@ workflow ATAC {
   String cutadapt_docker = "cutadapt:1.0.0-4.4-1686752919"
   String samtools_docker = "samtools-dist-bwa:3.0.0"
   String upstools_docker = "upstools:1.0.0-2023.03.03-1704300311"
-  String snap_atac_docker = "snapatac2:1.0.9-2.6.3-1715865353"
+  String snap_atac_docker = "snapatac2:lk-PD-2738"
```

Comment:
Can we change the branch name in the docker path to the docker version?

Reply:
thank you! Just added the updated version